[PowerToys]Mouse without borders: Disable Easy mouse when the foreground application is in fullscreen mode - docs

dot-tb opened the pull request:
PowerToys linked PR : microsoft/PowerToys#39854

dot-tb committed:
Update docs for PowerToys.MouseWithoutBorders PR

alvinashcraft commented on hub/powertoys/mouse-without-borders.md:
```diff
@@ -74,6 +74,8 @@ To enable the service mode, run PowerToys in administrator mode and turn on the
 | Show clipboard and network status messages | Show clipboard activities and network status in system tray notifications. |
 | Easy Mouse | Use the mouse pointer to switch between machines at screen edges. Can also be configured to need to select **Shift** or **Control** to switch between machines. |
+| Disable easy mouse when an application is in fullscreen | Block easy mouse from switching machine when an application is running in fullscreen mode on the host machine. |
+| Ignored fullscreen applications | List of ignored executable that will let easy mouse switch machine even if they are running in fullscreen mode. |
```
```suggestion
| Ignored fullscreen applications | A list of ignored executables that will allow easy mouse to switch machines even if they are running in fullscreen mode. |
```

alvinashcraft commented on hub/powertoys/mouse-without-borders.md:
```diff
@@ -74,6 +74,8 @@ To enable the service mode, run PowerToys in administrator mode and turn on the
 | Block mouse at screen corners | Avoid accidentally switching machines when the mouse pointer is at screen corners. |
 | Show clipboard and network status messages | Show clipboard activities and network status in system tray notifications. |
 | Easy Mouse | Use the mouse pointer to switch between machines at screen edges. Can also be configured to need to select **Shift** or **Control** to switch between machines. |
+| Disable easy mouse when an application is in fullscreen | Block easy mouse from switching machine when an application is running in fullscreen mode on the host machine. |
```
```suggestion
| Disable easy mouse when an application is in fullscreen | Prevents easy mouse from switching machines when an application is running in fullscreen mode on the host machine. |
```

alvinashcraft reviewed (commented):
Looks good. I left a couple of minor suggestions. Thanks! I'll hold off on merging this until the feature PR is merged and released in a PT release.

dot-tb committed:
Update hub/powertoys/mouse-without-borders.md

Accepted suggestion for Ignored fullscreen applications setting.

Co-authored-by: Alvin Ashcraft <73072+alvinashcraft@users.noreply.github.com>

dot-tb committed:
Update hub/powertoys/mouse-without-borders.md

Accepted suggestion for disable easy mouse when an application is in fullscreen

Co-authored-by: Alvin Ashcraft <73072+alvinashcraft@users.noreply.github.com>

alvinashcraft reviewed (approved):
Thanks for the nudge about this PR. I missed this one. Looks good!
